marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadFunctionMetaDataCommand.java

```diff
@@ -0,0 +1,76 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.datamovement.ApplyTransformListener;
+import com.marklogic.client.datamovement.DataMovementManager;
+import com.marklogic.client.datamovement.QueryBatcher;
+import com.marklogic.client.document.ServerTransform;
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.HubConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class LoadFunctionMetaDataCommand extends AbstractCommand {
```

Comment:
I recommend a naming that doesn't use "Load" because nothing is being loaded from a user's perspective. Instead, function metadata XML documents are being generated based on libraries of module functions that have already been loaded. 

How about "GenerateFunctionMetadataCommand"? (I would also use "Metadata" instead of "MetaData" as the former is consistent with how the word is spelled in other parts of MarkLogic).